docs/source/release/v6.12.0/Workbench/New_features/37538.rst

```diff
@@ -0,0 +1,2 @@
+- Add `mantidworkbench` as an entry point to launch the mantidworkbench Conda package.
+- Configure the Conda `workbench` and `mantidworkbench` entry points to launch workbench with jemalloc on Linux.
```

Comment:
RST needs double backticks for code formatting:
```suggestion
- Add ``mantidworkbench`` as an entry point to launch the mantidworkbench Conda package.
- Configure the Conda `workbench` and ``mantidworkbench`` entry points to launch workbench with jemalloc on Linux.
```

Reply:
I edited your suggestion to add double ticks around the workbench as well.